boards/arm/stm32f3_disco/dts.fixup

```diff
@@ -0,0 +1,13 @@
+/* This file is a temporary workaround for mapping of the generated information
```

Comment:
Rename this file to dts_fixup.h. See commit 8eb734c

Reply:
Done